crates/hc_sandbox/src/run.rs

```diff
@@ -201,18 +208,8 @@ async fn check_lair_running(stdout: tokio::process::ChildStdout) {
     let _ = r.await;
 }
 
-// TODO: Find a better way to confirm the child is running.
-async fn check_started(holochain: &mut Child) {
-    let started =
-        tokio::time::timeout(std::time::Duration::from_millis(20), holochain.wait()).await;
-    if let Ok(status) = started {
-        panic!("Holochain failed to start. status: {:?}", status);
-    }
-}
-
```

Comment:
Why is this no longer needed?

Reply:
It's no longer needed. Instead of trying to grab the exit status once (which we could now do with `try_wait`) we are relying on anything that goes wrong with the process leading to 

1. `reader.next_line().await` giving back an error when the child dies and stdout gets closed
2. loop ends
3. task ends
4. oneshot::Sender dropped
5. The receiver errors because the sender has dropped
6. Hit the `Err(_) => {` branch of the match above

I think it's more reliable this way because `check_started` gets a point in time and if holochain hasn't already exited then we miss that condition and end up with the `expect` that was giving a misleading error. This version is a lot more subtle but is detecting the child shutting down quite directly